src/Stormpath/Http/HttpClientRequestExecutor.php

```diff
@@ -91,4 +91,9 @@ private function addQueryString(array $queryString, RequestInterface $request)
         }
     }
 
+    public function getApiKey()
```

Comment:
We shouldn't need this if we move the apiKey to the dataStore. The apiKey could be contained in the Request object and use it from there.